components/TeammateSuggestions.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback, useState } from "react";
+import styled from "styled-components";
+import LinkedIn from "../assets/linkedin.svg";
+
+const TeammateSuggestions = ({ teammateSuggestions }) => {
+  return (
+    <>
+      <Title>Teammate Suggestions</Title>
+      <Suggestions>
+        {teammateSuggestions.hackerProfiles.map((suggestion) => (
+          <>
+            {suggestion.firstName ? (
+              <Suggestion key={suggestion.userId}>
+                {
+                  suggestion.portfolioUrl ? (
+                  <Header>
+                    <PortfolioLink>
+                      <a href={suggestion.portfolioUrl} target="_blank">
+                        <img src={LinkedIn} />
+                      </a>
+                    </PortfolioLink>
+                    <NameHeader>
+                      <Name>
+                        {suggestion.firstName} {suggestion.lastName}
+                      </Name>
+                      <Subheading>
+                        {suggestion.year ? suggestion.year : "year unavailable"}
+                      </Subheading>
+                    </NameHeader>
+                  </Header>
+                  ) : (
+                    <>
+                      <Name>
+                        {suggestion.firstName} {suggestion.lastName}
+                      </Name>
+                      <Subheading>
+                        {suggestion.year ? suggestion.year : "year unavailable"}
+                      </Subheading>
+                    </>
+                  )
+                }
+
+                
+                <Subheading>
+                  {suggestion.major ? suggestion.major : "major unavailable"}
```

Comment:
Can we show just `submitted` hackers so we won't see the `x unavailable` case?

Reply:
Yeah, the request filters based on who's confirmed, but I can change it to submitted if that's better

And sorry, yeah there's still a bunch I need to get done, but I'll make those other changes